src/codec_svt.c

```diff
@@ -205,6 +205,11 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         }
 #endif
 
+#if SVT_AV1_CHECK_VERSION(3, 0, 0)
+        svt_config->lossless = quantizer == AVIF_QUANTIZER_LOSSLESS;
+        svt_config->avif = (addImageFlags & AVIF_ADD_IMAGE_FLAG_SINGLE) != 0;
```

Comment:
I verified that src/codec_aom.c sets the equivalent fields under the same conditions. So these two lines are correct.

Reply:
Thanks!

Reply:
As described in https://gitlab.com/AOMediaCodec/SVT-AV1/-/issues/2245, `avif` mode cannot be used in libavif for now.